javascript/javascript-6-object-literals.md

```diff
@@ -28,16 +28,10 @@ About 60 minutes
 - JavaScript's Object Literal syntax
 - The concept of unordered property-value pairs
 - How to access an Object Literal's data 
-	- With dot-notation 
-	- With bracket notation
+  - With dot-notation 
+  - With bracket notation
 - `Object.hasOwnProperty()`
 
-### Supplemental Materials
```

Comment:
Be sure to add a ### Materials section here with all assigned materials found in the outline. Unnassigned/optional can go in supplemental.

Reply:
Done